website/versioned_docs/version-0.24.0/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -421,7 +418,7 @@ gitops version
 
 ## Next steps
 
-In our following [Get Started document](../../getting-started/ui.mdx), we will walk you through logging into the GitOps Dashboard and deploying an application.
+In our following Get Started document, we walk you through logging into the GitOps Dashboard and deploying an application.
```

Comment:
same here, seems we are missing the getting started page

Reply:
Removed section